include/linux/cei_hw_id.h

```diff
@@ -0,0 +1,58 @@
+#ifndef __CEI_HW_ID_H
```

Comment:
@kholk is this file really needed?

Reply:
it's not, as I said this PR is not ready for merge.

Reply:
Oh sure, keep up the great work!